src/language/control-flow.md

```diff
@@ -1,306 +0,0 @@
----
```

Comment:
is there an HTTP redirect for people who get to this page from old links?

(not a JS-based redirector)

Reply:
I'm not sure, maybe @parlough or @domesticmouse can answer that

Reply:
Yes, you set it up in the firebase.json: https://github.com/dart-lang/site-www/pull/4780#discussion_r1176971293